DCO

```diff
@@ -0,0 +1,36 @@
+Developer Certificate of Origin
```

Comment:
Let's please add the legal files in separate PRs. We agreed on that a while ago and did it like that in other projects.

Reply:
moved this in a different PR https://github.com/giantswarm/microendpoint/pull/29